docs/api/flink/Function.md

```diff
@@ -56,6 +56,33 @@ Spark SQL example:
 SELECT ST_Buffer(polygondf.countyshape, 1)
 FROM polygondf
 ```
+## ST_YMin
```

Comment:
Please read the doc carefully. version is wrong. Example is not complete with input/output